js/main.js

```diff
@@ -194,4 +197,51 @@ newEntryForm.addEventListener('submit', function (event) {
   photo.setAttribute('src', 'images/placeholder-image-square.jpg');
   newEntryForm.reset();
   viewSwapper('entries');
+
+  renderOneEntry()
 });
+
+var olEntries = document.querySelector('ol');
+
+function renderEntry(entry) {
+  var li = document.createElement('li');
+
+  var row = document.createElement('div');
+  row.setAttribute('class', 'row new-entry');
+  li.appendChild(row);
+
+  var col = document.createElement('div');
+  col.setAttribute('class', 'column-half');
+  row.appendChild(col);
+
+  var img = document.createElement('img');
+  img.setAttribute('src', entry.url);
+  img.setAttribute('alt', entry.title);
+  img.setAttribute('class', 'entry-image');
+  col.appendChild(img);
+
+  col2 = document.createElement('div');
+  col2.setAttribute('class', 'column-half');
+  row.appendChild(col2);
+
+  h4Element = document.createElement('h4');
+  h4Element.setAttribute('class', 'entry-header');
+  h4Element.textContent = entry.title;
+  col2.appendChild(h4Element);
+
+  pElement = document.createElement('p');
+  pElement.textContent = entry.notes;
+  col2.appendChild(pElement);
+
+  olEntries.appendChild(li);
```

Comment:
You should not be appending within the function like this, you should return the DOM tree, and append it where the function is called 👍